File: Coding/Nicomachus Theorem/Squared_triangular_number2.py
# A formula based Python program to find sum
# of series with cubes of first n natural
# numbers

# A000537 		Sum of first n cubes; or n-th triangular number squared. 

# Returns the sum of series
def sumOfSeries(n):
	x = (n * (n + 1) / 2)
	return (int)(x * x)



# Driver Function


n = int(input())
print(sumOfSeries(n))

# Code Contributed by Mohit Gupta_OMG <(0_o)>

File: Coding/Nicomachus Theorem/Squared_triangular_number2.cpp
// A formula based C++ program to find sum
// of series with cubes of first n natural
// numbers

// A000537 		Sum of first n cubes; or n-th triangular number squared. 
#include <iostream>
using namespace std;

int sumOfSeries(int n)
{
	int x = (n * (n + 1) / 2);
	return x * x;
}

// Driver Function
int main()
{
	int n =0;
	cin >> n;
	cout << sumOfSeries(n) << endl;
	return 0;
}
